webapp/src/pages/_app.tsx

```diff
@@ -22,6 +22,18 @@ type AppPropsWithLayout = AppProps & {
 
 const DogmaApp = ({ Component, pageProps }: AppPropsWithLayout) => {
   const router = useRouter();
+  useEffect(() => {
+    // Next.js uses a path pattern to dynamically match the request path to a specific HTML file.
+    // For example, `/app/projects/[projectNames]/index.html' is generated to render `/app/projects/myProj`
+    // page. `FileService` serving the static resources for Central Dogma webapp does not understand the
+    // path pattern syntax in the folder names. If `/ap/projects/myProj` is requested, the FileService will try
+    // to find '/app/projects/myProj/index.html' as is, and 'index.html' is returned as a fallback. As a
+    // workaround, this triggers Next.js router to route to the desired page when a page is loaded for the
+    // first time.
+    if (router.asPath !== '/') {
+      Router.push(router.asPath);
```

Comment:
```suggestion
      router.push(router.asPath);
```

Reply:
`replace` looks better. Let me try to use it.